docs/workforce-management-framework/topics/con-Security-Strategy-for-Keycloak.adoc

```diff
@@ -1,27 +1,28 @@
 [id='Security-Strategy-Keycloak-{chapter}']
-= Security Strategy for {WFM-RC-NameShort} Keycloak
+= Security Strategy for {WFM-RC-NameShort} {WFM-RC-KeycloakName}
```

Comment:
{WFM-RC-KeycloakName} why not {KeycloakProductName}

Reply:
variable naming standard introduced by John.